src/ElectronicRegulator.cpp

```diff
@@ -662,161 +196,129 @@ double compute_feedforward(double pressure_setpoint, double hp) {
+        if(rawSpd<MAX_SPD && rawSpd>MIN_SPD){ //anti-windup
+            angle_errorInt+=e*dt;
+            rawSpd-=ki*angle_errorInt;
+        }
+        else{angle_errorInt=0;}
+        
+        if (isAngleUpdate) {
+            oldPosition = angle;
+        }
+        oldError=e;
 
-    //Compute Inner PID Servo loop
-    float rawSpd=-(kp*e+kd*(e-oldError)/float(dt));
-    
-    // double updated_i_term = ki*(angle_errorInt + e*dt);
-    if(rawSpd<MAX_SPD && rawSpd>MIN_SPD){ //anti-windup
-        angle_errorInt+=e*dt;
-        rawSpd-=ki*angle_errorInt;
-    }
-    else{angle_errorInt=0;}
-    
-    if (isAngleUpdate) {
-        oldPosition = angle;
-    }
-    oldError=e;
-
-    //Compute Outer Pressure Control Loop
-    pressure_e = LPpsi - pressure_setpoint;
-    p_buff->insert(t2/1.0e6, LPpsi);
-    double rawAngle = -( kp_outer*pressure_e + kd_outer*(p_buff->get_slope()) );
-    rawAngle += compute_feedforward(pressure_setpoint, HPpsi);
-    if(rawAngle<MAX_ANGLE && (rawAngle>MIN_ANGLE || pressure_errorInt<0)){
-        pressure_errorInt += pressure_e * dt;
-        rawAngle -= ki_outer * pressure_errorInt;
-    }
-    pressure_e_old = pressure_e;
+        //Compute Outer Pressure Control Loop
+        pressure_e = LPpsi - pressure_setpoint;
+        p_buff->insert(t2/1.0e6, LPpsi);
+        double rawAngle = -( kp_outer*pressure_e + kd_outer*(p_buff->get_slope()) );
+        rawAngle += compute_feedforward(pressure_setpoint, HPpsi);
+        if(rawAngle<MAX_ANGLE && (rawAngle>MIN_ANGLE || pressure_errorInt<0)){
+            pressure_errorInt += pressure_e * dt;
+            rawAngle -= ki_outer * pressure_errorInt;
+        }
+        pressure_e_old = pressure_e;
 
-    // Constrain angles and speeds
-    angle_setpoint = min(MAX_ANGLE, max(MIN_ANGLE, rawAngle));
-    speed = min(max(MIN_SPD,rawSpd),MAX_SPD);
+        // Constrain angles and speeds
+        angle_setpoint = min(MAX_ANGLE, max(MIN_ANGLE, rawAngle));
+        speed = min(max(MIN_SPD,rawSpd),MAX_SPD);
 
-    runMotor();
+        runMotor();
+    }
 
-    if (t2 - lastPrint > 1000) {
-        #ifndef USE_DASHBOARD
-        Serial.println( String(t2) + "\t"+ String(angle_setpoint) + "\t"+ String(pressure_setpoint) +"\t" + String(speed) + "\t" + String(motorAngle) + "\t" + String(HPpsi) + "\t" + String(LPpsi) + "\t" + String(InjectorPT) + "\t" + String(p_buff->get_slope()) + "\t" + String(pressure_errorInt) );     
-        #else
-        Comms::Packet packet = {.id = 1};
-        // Comms::packetAddFloat(&packet, sin(t2/1e6));
-        Comms::packetAddFloat(&packet, float(angle_setpoint));
-        Comms::packetAddFloat(&packet, pressure_setpoint);
-        Comms::packetAddFloat(&packet, float(speed));
-        Comms::packetAddFloat(&packet, motorAngle);
+    
+    //send data to AC
+    if (micros() - lastPrint > 1000) {
+        Comms::Packet packet = {.id = 85};
         Comms::packetAddFloat(&packet, HPpsi);
```

Comment:
P, I, D terms are all very useful in debugging
Same for angle setpoint

Add these to the packet too?